analytics/sensors/src/main/java/quarks/analytics/sensors/Deadtime.java

```diff
@@ -0,0 +1,98 @@
+package quarks.analytics.sensors;
+
+import java.util.Date;
+import java.util.Objects;
+import java.util.concurrent.TimeUnit;
+
+import quarks.function.Predicate;
+
+/**
+ * A generic "deadtime" {@link Predicate}.
```

Comment:
Saw this with another class, why is "deadtime" in quotes?

Maybe:

Predicate that discards values for a period of time after passing a value.